### PR TITLE
BIP to reserve nversion bits in blockheader

### PR DESCRIPTION
**Pull request by btcdrak**

### CODE REVIEW

File: bip-reservedbits.mediawiki [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+This specification does not reserve specific bits for specific purposes.
+
+==Backwards Compatibility==
+
+This proposal is backwards compatible, and does not require a soft fork to implement.
+
+==References==
+
+  - [[bip-0008.mediawiki|BIP8]]
```
Comment (jonathancross) [inline bip-reservedbits.mediawiki]: Please format using a `*` at the beginning of the line so this list renders correctly in mediawiki markup. Eg. code should be:
```
* [[bip-0008.mediawiki|BIP8]]
* [[bip-0009.mediawiki|BIP9]]
* [https://arxiv.org/pdf/1604.00575.pdf AsicBoost white paper]
* [https://github.com/BlockheaderNonce2/bitcoin/wiki nNonce2 proposal]
* [http://stratumprotocol.org/ Stratum protocol extension for version-rolling]
```

**Comment by bitkevin** [comment]
[poolin.com](https://poolin.com) already use `0x1FFFE000` as the default version mask for weeks, right now miners which enabled asicboost seems works well.

**Comment by timohanke** [comment]
ACK. 

This matches what is already in use by hardware and software. It makes sense to formalize it as a BIP.

**Comment by btcdrak** [comment]
@luke-jr Updated for your nits. Please can I have a BIP number assignment.

**Comment by odarboe** [comment]
@btcdrak check the word in the last sentence. "....bits for activation **sinalling**."

**Pull request merged**